src/status_im/android/platform.cljs

```diff
@@ -37,8 +37,7 @@
                                       :background-color "#7099e619"
                                       :border-radius    5
                                       :padding          4}
-                           :item     {:status-text {:color       styles/color-black
```

Comment:
It's not clear how this change is related to PR's objective, could you elaborate please, @tiensonqin?